fcli-core/fcli-common/src/main/resources/com/fortify/cli/common/i18n/FortifyCLIMessages.properties

```diff
@@ -82,15 +77,11 @@ fcli.action.on-invalid-version = Action to take if action schema version is not
 output.table.header.signatureStatus = Signature\nStatus
 
 # Generic, non command-specific output and query options
-arggroup.output.heading = Output options%n
-output = Specify output type and optional type arguments. Available output formats: ${COMPLETION-CANDIDATES}. \
-  The 'expr' output format takes a string containing '{property}' placeholders, other output \
-  formats take an optional, comma-separated list of properties to include in the output.
-style = Comma-separated list of style elements to apply to the selected output format. Allowed values: ${COMPLETION-CANDIDATES}.
-to-file = Write command output to the specified file instead of stdout.
-store = Store the JSON results of this command in a variable. Variables can be managed through \
-  the 'fcli util variable' command, and can be referenced using ::variable::[property] on any \
-  subsequent command. 
+arggroup.output.heading = Output options (see documentation link below)%n
```

Comment:
Please change to `Output options (also see documentation link below)%n` (add 'also')